plugins/cron/main.go

```diff
@@ -60,16 +60,17 @@ func (*cronPlugin) Run(ctx context.Context, config interface{}, srv v1.WerftServ
 			})
 		}
 
+		request := &v1.StartGitHubJobRequest{
+			Metadata: &v1.JobMetadata{
+				Owner:       "cron",
+				Annotations: annotations,
+				Trigger:     trigger,
+				Repository:  repo,
+			},
+			JobPath: task.JobPath,
+		}
 		_, err = c.AddFunc(task.Spec, func() {
-			_, err := srv.StartGitHubJob(ctx, &v1.StartGitHubJobRequest{
-				Metadata: &v1.JobMetadata{
-					Owner:       "cron",
-					Annotations: annotations,
-					Trigger:     trigger,
-					Repository:  repo,
-				},
-				JobPath: task.JobPath,
-			})
+			_, err := srv.StartGitHubJob(ctx, request)
```

Comment:
Frankly I don't understand why solves the issue. Could you elaborate a bit on why this works?

Reply:
The function that is passed as the second arg of `c.AddFunc()` is executed async. Then the loop has already finished and it gets the last value of  `task`. 
Maybe I am completely wrong but here's an example https://go.dev/play/p/HM7aueHpVQk

Reply:
That makes a lot of sense. Thank you for the fix :)